docs/guides/protect-page-login/flutter-web-redirect.mdx

```diff
@@ -0,0 +1,160 @@
+---
+id: flutter-web-redirect
+title: Protect a Page with Login in Flutter Web
+sidebar_label: Flutter Web application
+---
+
+```mdx-code-block
+import Teaser from './_common/teaser.mdx'
+import mp4 from './flutter_web_redirect/screencast.mp4'
+import webm from './flutter_web_redirect/screencast.webm'
+
+<Teaser
+  framework={<a href="https://flutter.dev/">Flutter</a>}
+  mp4={mp4}
+  webm={webm}
+/>
+```
+
+## Create Flutter Web App
+
+The below command will create a basic Flutter example application. We will just
+add to it an authentication layer to keep the guide simple.
+
+```shell-session
+flutter create myapp
+cd myapp
+```
+
+For HTTP requests we will use the [dio](https://github.com/flutterchina/dio)
+package and for managing environment variable we will use
+[flutter dotenv](https://github.com/java-james/flutter_dotenv).
+
+All the packages we use throughout this guide are listed in our `pubspec.yaml`.
+
+```mdx-code-block title="pubspec.yaml"
+import pubspec from '!!raw-loader!../../../code-examples/protect-page-login/flutter_web_redirect/pubspec.yaml'
+import CodeBlock from '@theme/CodeBlock'
+
+<CodeBlock language="yaml">{pubspec}</CodeBlock>
+```
+
+Next, create an Authentication service in our `lib/services` directory, which we
+will use to query the Ory APIs for session information.
+
+```shell-session
+touch lib/services/auth.dart
+```
+
+```mdx-code-block title="lib/services/auth.dart"
+import auth from '!!raw-loader!../../../code-examples/protect-page-login/flutter_web_redirect/lib/services/auth.dart'
+
+<CodeBlock language="dart">{auth}</CodeBlock>
+```
```

Comment:
Is this still about creating the app? Looks like it should be a new paragraph. Again, look at the Golang guide for structure inspirations.